app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollViewModel.java

```diff
@@ -9,6 +9,7 @@
 import ch.heigvd.pro.b04.android.Poll.Question.Question;
 
 public class PollViewModel extends ViewModel {
+    private MutableLiveData<List<Question>> questionToView = new MutableLiveData<>();
```

Comment:
Why not have a `MutableLiveData<Question>` instead ? Seems like you're only interested in the first value of the list